### PR TITLE
Add package description

### PR DESCRIPTION
**Pull request by laura-ding**

### CODE REVIEW

File: cmake/CPackage.cmake [hunk 1]
```diff
@@ -2,11 +2,12 @@
...
 macro(package is_one)
     message(STATUS "is_one is ${is_one}")
-    set(CPACK_PACKAGE_DESCRIPTION "nebula-graph")
-    set(CPACK_PACKAGE_CONTACT "nebula-graph")
+    set(CPACK_PACKAGE_DESCRIPTION "Nebula Graph")
+    set(CPACK_PACKAGE_CONTACT "Nebula Graph")
     set(CPACK_PACKAGE_VERSION ${NEBULA_BUILD_VERSION})
     set(CPACK_RPM_PACKAGE_LICENSE "Apache 2.0 + Common Clause 1.0")
     set(CPACK_PACKAGE_NAME nebula-graph)
+    set(CPACK_PACKAGE_VENDOR "vesoft")
```
Comment (Shylock-Hg) [inline cmake/CPackage.cmake:10]: `VEsoft Inc. Build` better

File: cmake/CPackage.cmake [hunk 2]
```diff
@@ -17,11 +18,15 @@ macro(package is_one)
...
     endif()
 
+    set(CPACK_DEBIAN_PACKAGE_MAINTAINER "vesoft")
+    set(CPACK_DEBIAN_PACKAGE_DESCRIPTION "A distributed, scalable, lightning-fast graph database")
     set(CPACK_DEB_COMPONENT_INSTALL YES)
     set(CPACK_DEBIAN_PACKAGE_ARCHITECTURE "amd64")
     set(CPACK_DEBIAN_PACKAGE_HOMEPAGE "https://github.com/vesoft-inc/nebula/releases")
     set(CPACK_DEBIAN_PACKAGE_CONTROL_EXTRA ${CMAKE_CURRENT_SOURCE_DIR}/package/postinst)
 
+    set(CPACK_RPM_PACKAGE_GROUP "vesoft")
```
Comment (Shylock-Hg) [inline cmake/CPackage.cmake:28]: `VEsoft Inc.`

File: cmake/CPackage.cmake [hunk 3]
```diff
@@ -17,11 +18,15 @@ macro(package is_one)
         set(CPACK_COMPONENTS_GROUPING ONE_PER_GROUP)
     endif()
 
+    set(CPACK_DEBIAN_PACKAGE_MAINTAINER "vesoft")
```
Comment (Shylock-Hg) [inline cmake/CPackage.cmake:21]: `VEsoft Inc.`

File: cmake/CPackage.cmake (hunk 1 above)
Comment (dutor) [reply cmake/CPackage.cmake:10]: agreed

File: cmake/CPackage.cmake (hunk 2 above)
Comment (dutor) [reply cmake/CPackage.cmake:28]: agreed

Comment (laura-ding) [reply cmake/CPackage.cmake:28]: The head of the file is `vesoft inc`. I think it's unified it as `vesoft inc`.

File: cmake/CPackage.cmake [hunk 4]
```diff
@@ -2,11 +2,12 @@
 
 macro(package is_one)
     message(STATUS "is_one is ${is_one}")
-    set(CPACK_PACKAGE_DESCRIPTION "nebula-graph")
-    set(CPACK_PACKAGE_CONTACT "nebula-graph")
+    set(CPACK_PACKAGE_DESCRIPTION "Nebula Graph")
+    set(CPACK_PACKAGE_CONTACT "Nebula Graph")
```
Comment (Shylock-Hg) [inline cmake/CPackage.cmake:6]: Could define constant instead of hard code?

**Pull request merged**